bigquery/snippets/job/integration_test.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package job demonstrates interactions with BigQuery's job resources, which
+// allow for execution of multiple kinds of workloads including queries, loads,
+// and data extracts.
+package job
+
+import (
+	"cloud.google.com/go/bigquery"
+	"cloud.google.com/go/storage"
+	"context"
+	"fmt"
+	"github.com/GoogleCloudPlatform/golang-samples/bigquery/snippets/bqtestutil"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"google.golang.org/api/iterator"
+	"io/ioutil"
+	"testing"
+	"time"
+)
+
+func TestAll(t *testing.T) {
```

Comment:
`TestAll` sounds like it would be the only test. Anything more specific?

Reply:
Renamed tests to more specific identifiers.